src/main/catalog-sources/index.ts

```diff
@@ -0,0 +1 @@
+export { KubeconfigSyncManager } from "./kubeconfig-sync.source";
```

Comment:
I think we don't need `.source` in the filename. Folder name is `catalog-sources`.

Reply:
sounds good